src/rapid_pro_to_engagement_db/rapid_pro_to_engagement_db.py

```diff
@@ -200,6 +200,13 @@ def sync_rapid_pro_to_engagement_db(rapid_pro, engagement_db, uuid_table, rapid_
                 continue
 
             # De-identify the contact's full urn.
+            if run.contact.uuid not in contacts_lut:
+                log.warning(f"Found a run from a contact that isn't present in the contacts export; skipping. "
```

Comment:
assert if UUID is deleted so that we can monitor any other case.

Reply:
Unfortunately I don't think we can do this easily, as runs only contain the rapid pro contact uuid, which doesn't get marked as deleted. The only way to auto-detect is to download all the messages too, search those for uuids that match those not in contacts, then check that the urn starts with deleted. Do you think we should do that?

Reply:
I see - let monitor the cases and reconsider doing this if we get other cases.